DDG4/src/Geant4PhysicsList.cpp

```diff
@@ -374,6 +375,17 @@ G4VUserPhysicsList* Geant4PhysicsListActionSequence::extensionList()    {
   return physics;
 }
 
+G4VUserPhysicsList* Geant4PhysicsListActionSequence::activateCeleritas()    {
```

Comment:
This should probably be enabled via a plugin, like other modifications of the physics list?